src/viz-supervision-revocation/VizSupervisionRevocation.js

```diff
@@ -130,31 +125,13 @@ export default function VizSupervisionRevocationContainer({
   if (!month) {
     return null;
   }
-  const currentMonthData = new Map();
-
-  if (dimension === DIMENSION_KEYS.total) {
-    const monthlyTotals = monthlyData
-      .get(month)
-      .find(recordIsTotalByDimension(dimension));
-
-    currentMonthData.set(TOTAL_KEY, splitByViolationType(monthlyTotals));
-  } else {
-    monthlyData
-      .get(month)
-      .filter(recordIsTotalByDimension(dimension))
-      .sort((a, b) =>
-        demographicsAscending(
-          a[DIMENSION_DATA_KEYS[dimension]],
-          b[DIMENSION_DATA_KEYS[dimension]]
-        )
-      )
```

Comment:
Did we lose the `sort` in the refactor?

Reply:
we no longer need it, because we now follow the category ordering in their respective Map objects